plugins/discourse-local-dates/spec/system/local_dates_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+describe "Local dates", type: :system, js: true do
+  fab!(:topic) { Fabricate(:topic) }
+    fab!(:user) { Fabricate(:user) }
+    fab!(:post) { Fabricate(:post_with_local_dates, topic: topic) }
+
+    it "renders local dates and date ranges correctly" do
```

Comment:
For this to work on the CI, I think you will need to set `user.user_option.timezone` to `"Asia/Singapore"`